src/screens/Forum/ForumSubcategoryPostsScreen.js

```diff
@@ -42,32 +49,18 @@ export default class ForumSubcategoryPostsScreen extends React.Component {
       }
     });
 
+    const { postLimit } = this.state;
+
     /* Only query posts w/ categoryID matching categoryID passed in from navigation */
     this.unsubscribeFromFirestore = firestore().collection('forum_posts')
```

Comment:
Where you getting this error? 
![image](https://user-images.githubusercontent.com/14829068/83473915-905b0300-a43f-11ea-9437-10605e744625.png)
If so don't be afraid to create the index! This might fix the problem you showed me before. 
(You can console.log the error message to make it easy to copy and paste)